### PR TITLE
[enterprise-4.15] OSDOCS-9369-RN: add RN entry

### PR DESCRIPTION
**Pull request by jldohmann**
Version(s): 4.15

Issue: https://issues.redhat.com/browse/OSDOCS-9369

Link to docs preview:

- feature entry: https://71264--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-15-release-notes#ocp-4-15-mco-mcn-status
- TP table (last one): https://71264--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-15-release-notes#ocp-4-15-technology-preview-tables

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by jldohmann** [comment]
@sergiordlr can you PTAL at the following docs PR for QE ack? This PR adds the release note entries for more detailed MCO state reporting. thank you!

File: release_notes/ocp-4-15-release-notes.adoc [hunk 1]
```diff
@@ -512,6 +512,11 @@ If the following conditions are true, the feature is enabled by default:
 [id="ocp-4-15-machine-config-operator"]
 === Machine Config Operator
 
+[id="ocp-4-15-mco-mcn-status"]
+==== Improved MCO state reporting by node (Technology Preview)
+
+With this release, you can now monitor updates for individual nodes as a Technology Preview. For more information, see xref:../post_installation_configuration/machine-configuration-tasks.adoc#checking-mco-node-status[Checking machine config node status].
```
Comment (tmalove) [inline release_notes/ocp-4-15-release-notes.adoc:518]:
```suggestion
With this release, you can monitor updates for individual nodes as a Technology Preview. For more information, see xref:../post_installation_configuration/machine-configuration-tasks.adoc#checking-mco-node-status[Checking machine config node status].
```

**Review by tmalove — APPROVED** [review]

**Comment by sergiordlr** [comment]
Hello!  It looks good to me.

Thanks!

**Pull request merged**